src/constants/data.ts

```diff
@@ -171,7 +171,7 @@ export type Result = typeof RESULTS[number]
 export const FeedbackType = {
   discipline: 'discipline',
   leaf_discipline: 'trainingset',
-  document: 'document',
+  vocabularyItem: 'document',
```

Comment:
Is there a reason for not renaming the string here?

Reply:
yea I am quite sure it's api related. not 100% sure tho

Reply:
Yes, I think it should stay.